MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -58,6 +59,7 @@ bool LayeredVolume::createRasterLayers(const MeshLib::Mesh &mesh,
 	this->_minimum_thickness = minimum_thickness;
 	_nodes = MeshLib::copyNodeVector(bottom->getNodes());
 	_elements = MeshLib::copyElementVector(bottom->getElements(), _nodes);
+	_materials.insert(_materials.begin(), _elements.size(), 0); 
```

Comment:
The `_materials` size after this line will grow by number of stored `_elements`; is it correct?
Especially the elements in materials vector will be shifted.